src/middleware/issueDetails.middleware.js

```
@@ -306,7 +280,7 @@ export const getIssueDetails = renderTemplate({
 
 /* eslint-disable no-return-assign */
 const zeroEntityCount = (req) => req.entityCount = { entity_count: 0 }
-const zeroIssueEntitiesCount = (req) => req.issueEntitiesCount = 0
+const zeroIssueEntitiesCount = (req) => req.issueEntitiesCount = { count: 0 }
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)